src/app/modules/highlight/top-highlight-cards/top-highlight-cards.component.html

```diff
@@ -16,32 +16,29 @@
    If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
 
  -->
-<div class="highlights">
 
+<div class="highlights">
   <div class="row">
     <div class="col-12">
       <h2>Highlighted projects</h2>
-
       <ng-container *ngIf="highlights && highlights.length > 0; else noHighlights">
         <p class="large">Our top {{highlights.length}} projects</p>
       </ng-container>
       <ng-template #noHighlights>
         <p class="large">Currently there are no highlighted projects</p>
       </ng-template>
-
     </div>
   </div>
-
   <div class="row">
-    <div *ngFor="let highlight of highlights" class="col-md-12 col-lg-4">
+    <div *ngFor="let highlight of highlights" class="col-md-12 col-lg-4 mb-4">
       <div class="card" routerLink="/project/details/{{highlight.projectId}}">
         <div class="card-title">
           <div class="icon"><img src="assets/images/code.svg" alt="" loading="lazy"></div>
           <h3>{{highlight.project.name}}</h3>
         </div>
-        <p class="large">{{highlight.description | striphtml}}</p>
+        <!--This should be changed to the highlight descriptions-->
+        <p class="large">{{highlight.project.shortDescription  | striphtml}}</p>
```

Comment:
Actually I was wrong, in this PR the highlight already has a description. The PR was merged earlier this sprint. Keep it at highlight.description.